app/src/main/dist/bin/crate

```diff
@@ -112,21 +112,22 @@ launch_service()
 {
     pidpath=$1
     daemonized=$2
-    props=$3
+    javaProperties=$3
+    props=$4
     es_parms="-Dcrate"
```

Comment:
afaik this must also be changed to -E but I'm not sure if this is valid and needed (what was this for anyway?)